lib/asciidoctor-pdf/converter.rb

```diff
@@ -2043,7 +2041,9 @@ def convert_inline_icon node
       end
       begin
         # TODO support rotate and flip attributes
-        %(<font name="#{icon_set}"#{size_attr}>#{::Prawn::Icon::FontData.load(self, icon_set).unicode icon_name}</font>)
+        key = [icon_set, icon_name].join('-')
+        data = make_icon(key).format_hash
```

Comment:
is make_icon coming from Prawn::Icon?

Reply:
Yes it is (see [here](https://github.com/jessedoyle/prawn-icon/blob/97c100edfedd9135b3d00f52a6286a2935abf64e/lib/prawn/icon/interface.rb#L84-L91)). 

`make_icon` is a public method in `Prawn::Icon`'s interface that instantiates a new icon instance without immediately rendering it to the document. 

The method also performs the mapping of FontAwesome 4 keys to FontAwesome 5 keys for compatibility.

Reply:
:+1: